modules/oadp-ibm-power-test-support.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-ibm-power-test-matrix_{context}"]
+= OADP support for target backup locations using IBM Power Systems
+
+IBM Power Systems ("cloud systems") running on OCP 4.11 and OADP 1.1.0 was tested successfully aganst IIBM Cloud and AWS S3 backup location targets. Although this combination was tested only against these backup location targets, Red Hat supports running IBM Z with OCP 4.11 and OADP 1.1.0 againt all non-AWS S3 backup location targets.
```

Comment:
*IBM cloud object storage
*against (typo)

Reply:
@RichardHoch please note also the comment about the IBM Cloud - should be [IBM cloud object storage](https://www.ibm.com/cloud/object-storage) (no need to put the link, just correct the name :))